_dev/src/store/modules/app/actions.ts

```diff
@@ -130,7 +130,7 @@ export default {
         throw new HttpClientError(response.statusText, response.status);
       }
       const json = await response.json();
-
+      commit(MutationsTypes.SET_MODULE_DETAILS, json);
```

Comment:
Something is wrong here. This action can be triggered for `eventbus` and `psxmarketingwithgoogle`, so I can imagine that in some case, we end with the hooks of `eventbus`?